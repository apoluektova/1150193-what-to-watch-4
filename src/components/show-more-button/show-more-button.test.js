import React from "react";
import renderer from "react-test-renderer";
import ShowMoreButton from "./show-more-button.jsx";

it(`Show More button should render correctly`, () => {
  const tree = renderer
     .create(<ShowMoreButton
       handleShowMoreButtonClick={() => {}}
     />)
     .toJSON();

  expect(tree).toMatchSnapshot();
});
