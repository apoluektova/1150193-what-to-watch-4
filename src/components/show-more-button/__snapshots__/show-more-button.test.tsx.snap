// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show More button should render correctly 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Show more
</button>
`;
